### PR TITLE
Better Documentation for New Contributors

### PR DESCRIPTION
**Pull request by narendasan**
Making it easier for new contributors to get started

### CODE REVIEW

**Comment by nmagerko** [comment]
Sure. I assumed people would be using something like Workbench or some other GUI program that turns this into a click or two, but having this is reasonable too.

**Comment by nmagerko** [comment]
Not really a review, but do you need backticks on your `CREATE` statement? #33 makes it sound like that'd be necessary.

**Comment by narendasan** [comment]
Yeah I would add the backticks, but I didn't know if you had a solution for it so I didn't add it. Will add it now

**Comment by nmagerko** [comment]
Re-titled because I think this is a good place to put something like a `CONTRIBUTING.md` along with any new documentation that will help external contributors better understand what's available to work on, and how to go about working on it.

**Comment by narendasan** [comment]
@akuznets0v Can you review this?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+After you've made the appropriate changes, open a new [GitHub pull request](https://help.github.com/articles/about-pull-requests/).
+Be sure to review the guidelines below!
+
+#### How Do I Submit a (Good) Pull Request?
+
+All changes must be reviewed before being merged into the codebase, and some
+additional changes may be requested before the pull request is accepted. In order
+to make this process as easy as possible, be sure to do the following:
+
+* **Select your merge target** to be `staging` or lower, never `master`
```
Comment (redsn0w422) [inline CONTRIBUTING.md]: The "or lower" clause is a little confusing for me, do you mean branches that have been branched off of staging?

Comment (nmagerko) [inline CONTRIBUTING.md]: Yeah that's what I meant. Do you think I should just explicitly say "or branches that have been branched off of `staging`" instead?

**Pull request merged**